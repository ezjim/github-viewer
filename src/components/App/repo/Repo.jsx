import React from 'react';
import PropTypes from 'prop-types';

const Repo = ({ name, avatar_url }) => (
  <section>
    <img src={avatar_url} alt={name} />
    <p>{name}</p>
  </section>
);

Repo.propTypes = {
  name: PropTypes.string.isRequired,
  avatar_url: PropTypes.string.isRequired
};

export default Repo; 
