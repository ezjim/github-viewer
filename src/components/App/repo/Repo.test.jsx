import React from 'react';
import { shallow } from 'enzyme';
import Repo from './Repo';

describe('Repo component', () => {
  it('renders Repo', () => {
    const wrapper = shallow(<Repo 
      name="ezjim"
      html_url="https://github.com/ezjim"/>);
    expect(wrapper).toMatchSnapshot();
  });
});
