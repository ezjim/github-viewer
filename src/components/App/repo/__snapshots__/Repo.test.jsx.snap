// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Repo component renders Repo 1`] = `
<li>
  <p>
    ezjim
  </p>
  <p>
    https://github.com/ezjim
  </p>
</li>
`;
